src/sentry/static/sentry/app/views/settings/incidentRules/ruleForm/index.tsx

```diff
@@ -350,7 +419,7 @@ class RuleFormContainer extends AsyncComponent<Props, State> {
     // track of the list of triggers (and actions within triggers)
     try {
       addLoadingMessage();
-      const resp = await addOrUpdateRule(
+      const [resp, , xhr] = await addOrUpdateRule(
```

Comment:
wait... This is a thing?

Reply:
@scttcper Hahah, yes it's a rarely used feature of our request client.